extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -406,6 +406,7 @@ class QueueProcessor extends EventEmitter {
             groupId,
             concurrency: this.repConfig.queueProcessor.concurrency,
             queueProcessor: queueProcessorFunc,
+            logConsumerMetricsIntervalS: this.repConfig.queueProcessor.logConsumerMetricsIntervalS,
```

Comment:
Should also expose the metrics in Prometheus like we do in 7.x =>
https://github.com/scality/backbeat/blob/57fc2357d1bc9fe477b0f38f012f2e3633f68f70/extensions/replication/queueProcessor/QueueProcessor.js#L574-L580

We might want to do that only when `logConsumerMetricsIntervalS` is set (it's only set in federation, not zkop)